authors/apps/profiles/serializers.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import serializers
+
+from .models import Profile
+
+
+class ProfileSerializer(serializers.ModelSerializer):
+    username = serializers.CharField(source='user.username')
+    bio = serializers.CharField(allow_blank=True, required=False)
+    image = serializers.SerializerMethodField()
+
```

Comment:
When updating the user profile data. The image is not updated unless you change the above serializer method to `image = serializers.SerializerMethodField` in other words brackets should be removed. Kindly try testing with **http://127.0.0.1:8000/api/user/**   when updating an image to see what I mean.